bundles/org.openhab.binding.sonnen/src/main/java/org/openhab/binding/sonnen/internal/SonnenHandler.java

```diff
@@ -93,6 +102,8 @@ public void initialize() {
                 }
             }
         });
+
+        updateStatus(ThingStatus.UNKNOWN);
```

Comment:
This must be set before starting the asynchronous task (like before).

Reply:
ok